### DISCUSSION TITLE
Unable to create Organization

### DISCUSSION THREAD

**Discussion by niteshg0** [community/community#148129] [New to GitHub]
### Issue: Unable to Verify Captcha When Creating a New Organization on GitHub

When attempting to create a new organization on GitHub, the following error message appears:

```
Unable to verify your captcha answer. Please try again or visit https://docs.github.com/articles/troubleshooting-connectivity-problems/#troubleshooting-the-captcha for troubleshooting information.
```

There is no captcha. It shows a Green tick.

**Comment by ironcito** [comment]
Obvious question: did you solve the captcha? Did you visit the page mentioned in the error message and try the solutions provided there? Are you able to solve captchas on other websites?

**Reply by niteshg0** [reply]
There is no captcha. It shows a Green tick.

**Comment by nishanmudalige** [comment]
It's possible that extensions on the browser you're using is preventing the captcha from showing.

Try using another browser, or disable all extensions on the browser you're using and try again.

Are you using any VPN's when you access the site? If you are, try turning them off.

**Reply by aflacake** [reply]
Remove/nonactive the extension, it usually has an effect. Then reactivate it if successful.

**Comment by ghostinhershell** [comment]
Hey @niteshg0, thanks so much for reaching out!I see some great answers here, are you still running into this issue?
